sdk/keyvault/azkeys/client.go

```diff
@@ -73,7 +73,7 @@ type CreateKeyOptions struct {
 
 	// The attributes of a key managed by the key vault service.
 	KeyAttributes *KeyAttributes   `json:"attributes,omitempty"`
-	KeyOps        []*KeyOperations `json:"key_ops,omitempty"`
+	KeyOps        []*KeyOperation `json:"key_ops,omitempty"`
```

Comment:
The only place we exposed "KeyOps" was in the `JSONWebKey` class (struct). Everywhere else it was `KeyOperations`.

Reply:
In hindsight this was a mistake, IMO, but we already shipped like that. I think the original idea is for the JWK to more closely match its underlying JSON structure.

Reply:
I'm slightly confused by this statement, are you recommending anywhere that exposes a `[]*KeyOperations` the field should be called `KeyOperations`, and in the `JSONWebKey` struct it should be called `KeyOps` ?

Reply:
Checked out the apiview for .NET, understood now. Changes have been made.